p2p/module.go

```diff
@@ -0,0 +1,193 @@
+package p2p
+
+import (
+	"errors"
+	"log"
+	"net"
+	"pocket/p2p/types"
+	"pocket/shared/config"
+	"pocket/shared/modules"
+	"pocket/shared/types"
+	"sync"
+
+	"go.uber.org/atomic"
+	"google.golang.org/protobuf/types/known/anypb"
+)
+
+var (
+	MaxInbound           uint = 128
+	MaxOutbound          uint = 128
+	WireByteHeaderLength int  = 9
+	ReadBufferSize       int  = (1024 * 4)
+	WriteBufferSize      int  = (1024 * 4)
+	ReadDeadlineMs       int  = 400
+)
+
+var (
```

Comment:
1. Just because of prototyping, they should be consts
2. They should be loaded from the config, at the time it was more convenient to use var/consts as presumed default config values.